tests/interactions/io_wrappers/test_tensor_audio_recorder.py

```diff
@@ -0,0 +1,103 @@
+import numpy as np
+import pytest
+import soundfile as sf
+import torch
+
+from ami.interactions.io_wrappers.tensor_audio_recorder import TensorAudioRecorder
+
+CHANNELS = 2
+SAMPLE_SIZE = 16080
+
+
+class TestTensorAudioRecoder:
+    TEST_TENSOR = torch.rand(CHANNELS, SAMPLE_SIZE) * 2 - 1
+
+    @pytest.fixture
+    def output_dir(self, tmp_path):
+        return tmp_path / "audio_output"
+
+    @pytest.fixture
+    def recorder(self, output_dir):
+        recoder = TensorAudioRecorder(
+            output_dir,
+            sample_rate=16000,
+            channel_size=CHANNELS,
+        )
+        recoder.setup()
+        yield recoder
+        recoder.teardown()
+
+    def test_open_file_writer(self, recorder: TensorAudioRecorder):
+        assert recorder.file_path.exists()
+        assert not recorder.file_writer.closed
+
+    def test_wrap(self, recorder: TensorAudioRecorder):
+        result = recorder.wrap(self.TEST_TENSOR)
+        assert result is self.TEST_TENSOR
+
+    def test_recorded_audio(self, recorder: TensorAudioRecorder):
+        # Record multiple frames of audio
+        for _ in range(5):
+            recorder.wrap(self.TEST_TENSOR)
+        recorder.teardown()
+
+        # Read the recorded audio file
+        data, sample_rate = sf.read(recorder.file_path)
```

Comment:
SoundFileじゃなくtorchaudio使いたいってこと以外にも、`data`って変数名だと何を指してるかちょっと曖昧なのが気になった
`waveform`とか`wav`とかみたいに、波形であることを強調したさある
```suggestion
        waveform, sample_rate = torchaudio.load(recorder.file_path)
```